states.c

```diff
@@ -21,75 +22,42 @@
 #include <linux/spinlock.h>
 #include <linux/idr.h>
 #include <linux/ratelimit.h>
+#include <linux/hashtable.h>
+#include <linux/stringhash.h>
 
-#include <lualib.h>
+#include "lua/lualib.h"
 
 #include "luautil.h"
-#include "xt_lua.h"
 #include "states.h"
-#include "netlink.h"
-#include "kpi_compat.h"
-
-#ifndef NFLUA_SETPAUSE
-#define NFLUA_SETPAUSE	100
-#endif /* NFLUA_SETPAUSE */
-
-extern int luaopen_memory(lua_State *);
-extern int luaopen_conn(lua_State *);
-extern int luaopen_netlink(lua_State *);
-extern int luaopen_packet(lua_State *);
-extern int luaopen_timer(lua_State *);
-
-static const luaL_Reg libs[] = {
-	{"memory", luaopen_memory},
-	{"conn", luaopen_conn},
-	{"netlink", luaopen_netlink},
-	{"packet", luaopen_packet},
-	{"timer", luaopen_timer},
-	{NULL, NULL}
-};
+
+#ifndef KLUA_SETPAUSE
+#define KLUA_SETPAUSE	100
+#endif /* KLUA_SETPAUSE */
+
```

Comment:
This variable still here in order to provide the unique meta state for the whole API